eclass/multilib.eclass

```diff
@@ -415,6 +415,14 @@ multilib_env() {
 			: ${MULTILIB_ABIS=lp64d lp64 ilp32d ilp32}
 			: ${DEFAULT_ABI=lp64d}
 
+			# the default abi is set to the 1-level libdir default
+
+			local __libdir_riscvdefaultabi_variable="LIBDIR_${DEFAULT_ABI}"
+			local __libdir_riscvdefaultabi=${!__libdir_riscvdefaultabi_variable}
+			export ${__libdir_riscvdefaultabi_variable}=${__libdir_riscvdefaultabi:-lib64}
```

Comment:
This 3-lines solution works, but it is alternatively possible to use 1-line solution:
```
eval "export LIBDIR_${DEFAULT_ABI}=\${LIBDIR_${DEFAULT_ABI}:-lib64}"
```

Reply:
No thanks, I'm going for readability (and the hope that someone else will understand it in the future) there.